<ui version="4.0" >
 <class>DAQSettingsDialog</class>
 <widget class="QDialog" name="DAQSettingsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>DAQ Settings</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label_1" >
       <property name="text" >
        <string>Number of &amp;Channels:</string>
       </property>
       <property name="buddy" >
        <cstring>numChannels</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QSpinBox" name="numChannels" >
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="maximum" >
        <number>8</number>
       </property>
       <property name="value" >
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_10" >
       <property name="text" >
        <string>&amp;Sampling Rate</string>
       </property>
       <property name="buddy" >
        <cstring>samplingRate</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="samplingRate" />
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>&amp;Length In Max Voltage</string>
       </property>
       <property name="buddy" >
        <cstring>maxV1</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="maxV1" />
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>&amp;Force In Max Voltage</string>
       </property>
       <property name="buddy" >
        <cstring>maxV2</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QLineEdit" name="maxV2" />
     </item>
     <item row="4" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Aux &amp;1 Max Voltage</string>
       </property>
       <property name="buddy" >
        <cstring>maxV3</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QLineEdit" name="maxV3" />
     </item>
     <item row="5" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Aux &amp;2 Max Voltage</string>
       </property>
       <property name="buddy" >
        <cstring>maxV4</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1" >
      <widget class="QLineEdit" name="maxV4" />
     </item>
     <item row="6" column="0" >
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>Aux &amp;3 Max Voltage</string>
       </property>
       <property name="buddy" >
        <cstring>maxV5</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1" >
      <widget class="QLineEdit" name="maxV5" />
     </item>
     <item rowspan="2" row="7" column="0" >
      <widget class="QLabel" name="label_7" >
       <property name="text" >
        <string>Aux &amp;4 Max Voltage</string>
       </property>
       <property name="buddy" >
        <cstring>maxV6</cstring>
       </property>
      </widget>
     </item>
     <item rowspan="2" row="7" column="1" >
      <widget class="QLineEdit" name="maxV6" />
     </item>
     <item row="9" column="0" >
      <widget class="QLabel" name="label_8" >
       <property name="text" >
        <string>Aux &amp;5 Max Voltage</string>
       </property>
       <property name="buddy" >
        <cstring>maxV7</cstring>
       </property>
      </widget>
     </item>
     <item row="9" column="1" >
      <widget class="QLineEdit" name="maxV7" />
     </item>
     <item row="10" column="0" >
      <widget class="QLabel" name="label_9" >
       <property name="text" >
        <string>Aux &amp;6 Max Voltage</string>
       </property>
       <property name="buddy" >
        <cstring>maxV8</cstring>
       </property>
      </widget>
     </item>
     <item row="10" column="1" >
      <widget class="QLineEdit" name="maxV8" />
     </item>
     <item row="0" column="2" >
      <widget class="QPushButton" name="fgColor" >
       <property name="text" >
        <string>Foreground</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QPushButton" name="bgColor" >
       <property name="text" >
        <string>Background</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" >
      <widget class="QPushButton" name="color1" >
       <property name="text" >
        <string>Length In</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2" >
      <widget class="QPushButton" name="color2" >
       <property name="text" >
        <string>Force In</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" >
      <widget class="QPushButton" name="color3" >
       <property name="text" >
        <string>Aux 1</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2" >
      <widget class="QPushButton" name="color4" >
       <property name="text" >
        <string>Aux 2</string>
       </property>
      </widget>
     </item>
     <item rowspan="2" row="6" column="2" >
      <widget class="QPushButton" name="color5" >
       <property name="text" >
        <string>Aux 3</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2" >
      <widget class="QPushButton" name="color6" >
       <property name="text" >
        <string>Aux 4</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2" >
      <widget class="QPushButton" name="color7" >
       <property name="text" >
        <string>Aux 5</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2" >
      <widget class="QPushButton" name="color8" >
       <property name="text" >
        <string>Aux 6</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>numChannels</tabstop>
  <tabstop>fgColor</tabstop>
  <tabstop>samplingRate</tabstop>
  <tabstop>bgColor</tabstop>
  <tabstop>maxV1</tabstop>
  <tabstop>color1</tabstop>
  <tabstop>maxV2</tabstop>
  <tabstop>color2</tabstop>
  <tabstop>maxV3</tabstop>
  <tabstop>color3</tabstop>
  <tabstop>maxV4</tabstop>
  <tabstop>color4</tabstop>
  <tabstop>maxV5</tabstop>
  <tabstop>color5</tabstop>
  <tabstop>maxV6</tabstop>
  <tabstop>color6</tabstop>
  <tabstop>maxV7</tabstop>
  <tabstop>color7</tabstop>
  <tabstop>maxV8</tabstop>
  <tabstop>color8</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
